test/__snapshots__/index.test.js.snap

```diff
@@ -701,6 +701,17 @@ exports[`fixtures build basic-multi-source with microbundle 8`] = `
 "
 `;
 
+exports[`fixtures build basic-multi-source-css with microbundle 1`] = `
+"Used script: microbundle
+
+Directory tree:
+
+basic-multi-source-css
+
+
+Error: No entry module found for \\"basic-multi-source-css\\""
+`;
+
```

Comment:
Looks like you didn't remove the empty directory that Git left after switching branches
```suggestion
```

Reply:
yes